### DISCUSSION TITLE
Why can't SmartThings multi-purpose sensor 3321-S show x/y/z axis?

### DISCUSSION THREAD

**Discussion by deep126** [Koenkk/zigbee2mqtt#9146] [Devices]
While I was converting my devices over from SmartThings I noticed that the 2015 multipurpose sensors lost the functionality to show its orientation on the x, y, and z axis. I even paired it back to SmartThings just to confirm. In the devices section on zigbee2mqtt.io it doesn't list the orientation exposes either so I initially thought z2m was choosing the wrong device because my sensors had those functions in SmartThings but not now in z2m. However I confirmed the model number and it just looks like it isn't set up in z2m for some reason. I am no where near capable to figure out a custom workaround without any assistance so I am wondering if no one else can help where do I start to figure this out myself?

**Comment by rsabol77** [comment]
Has this gone anywhere?
